templates/resources/organization_rule.md

```diff
@@ -0,0 +1,108 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "buildkite_organization_rule Resource - terraform-provider-buildkite"
+subcategory: ""
+description: |-
+  An Organization Rule allows specifying explicit rules between two Buildkite resources and the desired effect/action.
+  More information on organization rules can be found in the documentation https://buildkite.com/docs/pipelines/rules/overview.
+---
+
+# buildkite_organization_rule (Resource)
+
+~> Rules is a feature that is currently in development and enabled on an opt-in basis for early access. To have this enabled for your organization for utilizing this resource, please reach out to Buildkite's [Support Team](https://buildkite.com/support).
```

Comment:
This is to align with what is currently shown on the [docs](https://buildkite.com/docs/pipelines/rules) - and a pre-requisite before folks can use this resource for their organization.